practical-unit-testing-with-rtl/Exercise1-queries/problem/src/App.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+
+const Button = () => {
+  return <button> Click me to submit!</button>;
+};
+
+const TextInput = () => {
+  return (
+    <>
+      <label htmlFor="username">Username</label>
+      <input id="username" placeholder="Enter Username Here" />
```

Comment:
I’m guessing the point of this is to get folks to try `getByPlaceholderText()`.

In general we want to [avoid placeholders](https://www.deque.com/blog/accessible-forms-the-problem-with-placeholders/), especially when they’re redundant with the label. I would remove this and _maybe_ add a less contrived example if we can come up with a good use of `placeholder`.

Reply:
yeah, it was just to give the _option_ for folks to use that one.... im not sure its a big deal if this first intro exercise is kindof contrived? goal is just to understand the queries